task_DEV_1/task_DEV_1/Program.cs

```diff
@@ -8,24 +8,32 @@ static void Main(string[] args)
         {
             Console.WriteLine("Enter A Number :");
             int inputNumber = int.Parse(Console.ReadLine());
+            Console.WriteLine("Enter A Number :");
+            int secondInputNumber = int.Parse(Console.ReadLine());
 
-            if (inputNumber % 2 == 0 && inputNumber % 5 == 0)
-            {
-                Console.WriteLine("Tutti-Frutti");
-            }
-            else if (inputNumber % 2 == 0)
+            for (; inputNumber <= secondInputNumber; inputNumber++)
```

Comment:
Nice solution but I would recommend you don't change `inputNumber`. Because in such case you will have different value of one variable at the begining and at the end of your code and it can confuse somebody who will use your code. For example it the end of your code I want to write to console initial values of input parameters. How can I do this? Feel free to ask question if my explanation is not complitly clear.